### PR TITLE
Refactored templates to Octane style (angle bracket invocation, named args, this.args), allows setting HTML attributes using angle bracket invocation to the proper element

### PR DESCRIPTION
**Pull request by simonihmig**
- [x] enable skipped splattributes tests when https://github.com/kybishop/ember-popper/issues/98 is resolved

### CODE REVIEW

File: addon/templates/components/common/bs-modal.hbs [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+        @fade={{this.fade}}
+        @showModal={{this.showModal}}
+        @keyboard={{this.keyboard}}
+        @size={{@size}}
+        @backdropClose={{this.backdropClose}}
+        @inDom={{this.inDom}}
+        @paddingLeft={{this.paddingLeft}}
+        @paddingRight={{this.paddingRight}}
+        @centered={{bs-eq this.position "center"}}
+        ...attributes
```
Comment (jelhan) [inline addon/templates/components/common/bs-modal.hbs]: Not sure if you are aware of but using `...attributes` and innerHTML components together results in attributes being applied twice: https://ember-twiddle.com/74a339b46d1a87421927992e5a073d1f?openFiles=templates.application.hbs%2Ctemplates.components.my-component.hbs

Comment (simonihmig) [reply addon/templates/components/common/bs-modal.hbs]: Hm, yes, good catch! I think we could remove the default wrapper element here actually...

**Comment by simonihmig** [comment]
Failing tests due to https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/issues/67

**Comment by simonihmig** [comment]
This should be ready to merge from my PoV. @jelhan wanna have a 2nd look?

File: addon/templates/components/bs3/bs-accordion/body.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{{#component collapseComponent ariaRole="tabpanel" collapsed=collapsed class="panel-collapse"}}
-  <div class="panel-body {{class}}">
+{{#component this.collapseComponent ariaRole="tabpanel" collapsed=@collapsed class="panel-collapse"}}
```
Comment (jelhan) [inline addon/templates/components/bs3/bs-accordion/body.hbs]: Shouldn't this be `aria-role="tabpanel"` as the HTML attribute is name `aria-role` and not `ariaRole`?

File: addon/templates/components/common/bs-modal-simple.hbs [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
+      @showModal={{this.showModal}}
+      @keyboard={{this.keyboard}}
+      @size={{@size}}
+      @backdropClose={{this.backdropClose}}
+      @inDom={{this.inDom}}
+      @paddingLeft={{this.paddingLeft}}
+      @paddingRight={{this.paddingRight}}
+      @centered={{bs-eq this.position "center"}}
+      ...attributes
+      @class={{@class}}
```
Comment (jelhan) [inline addon/templates/components/common/bs-modal-simple.hbs]: Any reason for the order of `...attributes` and `@class` here? I thought properties won't be affected anyways by `...attributes`. If I would prefer an order of having all properties before attributes for readability.

File: addon/templates/components/bs3/bs-accordion/body.hbs (hunk 2 above)
Comment (simonihmig) [reply addon/templates/components/bs3/bs-accordion/body.hbs]: As the component helper does not allow to be used in a angle brackets like fashion (it's not a component), we cannot set HTML attributes in the way we could w/ angle bracket components. So this sets the `ariaRole` *property* of `Ember.Component` (see https://api.emberjs.com/ember/3.10/classes/Component/properties/ariaRole?anchor=ariaRole), that is mapped to the `role` attribute (through our beloved `attributeBindings`) internally.

File: addon/templates/components/common/bs-modal-simple.hbs (hunk 3 above)
Comment (simonihmig) [reply addon/templates/components/common/bs-modal-simple.hbs]: Hm, no. I probably have used `class={{@class}}` before, and wanted to make sure that when setting `@class` this takes precedence of the splattributes. But as this is a prop again, this does not make really sense, will fix it!

File: addon/templates/components/bs3/bs-accordion/body.hbs (hunk 2 above)
Comment (jelhan) [reply addon/templates/components/bs3/bs-accordion/body.hbs]: Thanks a lot for this detailed explanation. :+1:

**Pull request merged**